runtime/compiler/optimizer/J9RecognizedCallTransformer.cpp

```diff
@@ -1495,6 +1499,18 @@ void J9::RecognizedCallTransformer::transform(TR::TreeTop* treetop)
          case TR::java_lang_Math_min_L:
             processIntrinsicFunction(treetop, node, TR::lmin);
             break;
+         case TR::java_lang_Math_max_F:
```

Comment:
Does other platform supports `fmax/dmax/fmin/dmin` that adheres Java Specification ?  If not, I think this code will recognize and transform this to opcode on all platform.

Reply:
is should only transform it when the option `TR_DisableMaxMinOptimization` is not set. 

Other platforms define their own `f/dmin/maxEvaluators` so other platforms should still behave as before

Reply:
My comment was more in relationship with floating point max and min operation. All platforms (Including Z) had such floating point max and min evaluators but they were not recognizing floating point max and min operation, as those methods prior to this change was not checked in recognized call transformation. With this change, we would recognized java method, and then will convert the call to fmax/dmax/fmin/dmin node on all platform. I do not know if the code we generate on other platform would be OK with Java specification or not. This is something we should verify.

Reply:
it looks like the  current behaviour for Semeru on x86 and Power is the same as Z (the normal Java spec). testing the inlined versions now to see it they are different